src/compiler/path.ts

```diff
@@ -452,12 +452,9 @@ namespace ts {
      * Normalize path separators, converting `\` into `/`.
      */
     export function normalizeSlashes(path: string): string {
-        const index = path.indexOf("\\");
-        if (index === -1) {
-            return path;
-        }
-        backslashRegExp.lastIndex = index; // prime regex with known position
-        return path.replace(backslashRegExp, directorySeparator);
+        return path.indexOf("\\") !== -1

```

Comment:
Alternatively, use `===` and flip the branches.

Reply:
Yeah, I considered that, but opted to match the other functions that do `regex.test(s) ? s.replace(regex, "...") : s`. From my testing it was the same, I think.